lib/workerclient.go

```diff
@@ -53,6 +54,17 @@ const (
 	MaxWorkerState                  = 7
 )
 
+var validStateTransitionMap map[HeraWorkerStatus][]HeraWorkerStatus = map[HeraWorkerStatus][]HeraWorkerStatus{
+	wsUnset: {wsSchd, wsInit},
+	wsSchd:  {wsInit, wsUnset},
+	wsInit:  {wsSchd, wsAcpt, wsUnset},
+	wsAcpt:  {wsBusy},
+	wsBusy:  {wsWait, wsQuce, wsFnsh},
+	wsWait:  {wsQuce, wsFnsh},
+	wsFnsh:  {wsAcpt},
```

Comment:
From finished state a worker can go in scheduled state as well right ?
In my local i have seen the order going from fnsh -> schd -> acpt.

Reply:
It is valid transition. updated the map